packages/app/src/systems/Accounts/components/AccountConnectionInput/AccountConnectionInput.tsx

```diff
@@ -44,36 +47,40 @@ export const AccountConnectionInput = ({
               <Image width="20" height="20" src={networkImageUrl} />
               <Text>{networkName}</Text>
             </Flex>
-            <Flex>
-              {!currentAccount ? (
-                <Button
-                  onPress={onConnect}
-                  isLoading={isConnecting}
-                  css={styles.connectButton}
-                >
-                  Connect wallet
-                </Button>
-              ) : (
-                <Button
-                  isLoading={isConnecting}
-                  leftIcon={<Icon icon="Circle" color="red10" fill="red10" />}
-                  rightIcon={
-                    <IconButton
-                      icon="X"
-                      variant="link"
-                      color="gray"
-                      onPress={onDisconnect}
-                      aria-label="Disconnect account"
-                    />
-                  }
-                  variant="outlined"
-                  color="gray"
-                  css={styles.connectButton}
-                >
-                  {shortAddress(currentAccount)}
-                </Button>
-              )}
-            </Flex>
+            {!account?.address ? (
+              <Button
+                onPress={onConnect}
+                isLoading={isConnecting}
+                css={styles.connectButton}
+              >
+                Connect wallet
+              </Button>
+            ) : (
+              <Button
+                isLoading={isConnecting}
+                leftIcon={<Box css={styles.circle}>&nbsp;</Box>}
```

Comment:
We still need to add the Avatar here, but this could be done in another pr.

Reply:
yes let's do it together when upgrading `fuel-ui`